pack/ptc/dca_encounter.json

```diff
@@ -376,60 +376,73 @@
+        "encounter_position": 23,
+        "enemy_damage": 0,
         "enemy_evade": 2,
         "enemy_fight": 4,
         "enemy_horror": 2,
         "faction_code": "mythos",
         "health": 7,
+        "health_per_investigator": true,
         "illustrator": "Stephen Somers",
-        "name": "Hastur",
+        "is_unique": true,
+        "name": "Hastur",
         "pack_code": "dca",
         "position": 332,
         "quantity": 1,
         "subname": "The King in Yellow",
-        "text": "Cannot be damaged except by story card effects.<br/><b>Forced</b> - At the beginning of the enemy phase, if Hastur is ready: Hastur attacks each investigator, in player order . If Hastur's attack deals you horror in excess of your sanity, you take 1 damage, as well.<br/>[action]: Test [willpower] (X), where X is Hastur's fight value. If you succeed, exhaust Hastur. Any investigator may trigger this action.",
+        "text": "Cannot be damaged except by story card effects.\n<b>Forced</b> - At the beginning of the enemy phase, if Hastur is ready: Hastur attacks each investigator, in player order <i>(regardless of their location)</i>. If Hastur's attack deals you horror in excess of your sanity, you take 1 damage, as well.\n[action]: Test [willpower] (X), where X is Hastur's fight value. If you succeed, exhaust Hastur. Any investigator may trigger this action.",
         "traits": "Ancient One. Elite.",
         "type_code": "enemy"
     },
     {
         "code": "03333",
         "encounter_code": "dim_carcosa",
+        "encounter_position": 24,
+        "enemy_damage": 0,
         "enemy_evade": 3,
         "enemy_fight": 3,
         "enemy_horror": 2,
         "faction_code": "mythos",
         "health": 9,
+        "health_per_investigator": true,
         "illustrator": "Stephen Somers",
-        "name": "Hastur",
+        "is_unique": true,
+        "name": "Hastur",
         "pack_code": "dca",
         "position": 333,
         "quantity": 1,
         "subname": "Lord of Carcosa",
-        "text": "<b>Prey</b> - Most remaining sanity.<br/>Hunter. Massive.<br/>If Hastur's attack deals you horror in excess of your sanity, you take 1 damage, as well.<br/>While Hastur is being attacked or evaded by an investigator with no remaining sanity, treat each [elder_sign], \"+1,\" \"0,\" or \"-1\" token that investigator reveals as a [auto-fail] token.",
+        "text": "<b>Prey</b> - Most remaining sanity.\nHunter. Massive.\nIf Hastur's attack deals you horror in excess of your sanity, you take 1 damage, as well.\nWhile Hastur is being attacked or evaded by an investigator with no remaining sanity, treat each [elder_sign], \"+1,\" \"0,\" or \"-1\" token that investigator reveals as a [auto-fail] token.",
         "traits": "Ancient One. Elite.",
         "type_code": "enemy"
     },
     {
         "code": "03334",
         "encounter_code": "dim_carcosa",
+        "encounter_position": 25,
+        "enemy_damage": 0,
```

Comment:
Don't think you need to add values for 0. Just don't add the line at all - that's been fine previously.

Reply:
OK, I removed those lines.